tests/test_asvdb.py

```diff
@@ -181,11 +180,11 @@ def test_gitExtension():
     asvDir.cleanup()
 
 
+@pytest.mark.parametrize("asvDir", ["local", "s3"])
```

Comment:
I think you'll need to make the param name `asvDir` an arg to the test function too.

Reply:
Ah ya, forgot about that.